util/cuda_timer_t.cuh

```diff
@@ -0,0 +1,75 @@
+// Copyright Supranational LLC
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __SPPARK_UTIL_CUDA_TIMER_T_CUH__
+#define __SPPARK_UTIL_CUDA_TIMER_T_CUH__
+
+#include <iostream>
+#include <iomanip>
+
+class cuda_timer_t {
+    cudaEvent_t begin, end;
+public:
+    cuda_timer_t() : begin(nullptr), end(nullptr) {
```

Comment:
The common coding style for **this** project is to start multi-line functions with opening curly bracket on a separate line. Well, it's not really disqualifying, but it's not inappropriate to maintains uniform appearance...